@stellar/typescript-wallet-sdk/src/walletSdk/Server/index.ts

```diff
@@ -58,3 +66,23 @@ export const signChallengeTransaction = async ({
     networkPassphrase,
   };
 };
+
+/**
+ * Helper method for parsing a JSON string into an AnchorTransaction.
+ * @param {string} transaction - The json string of an anchor transaction.
+ * @returns {AnchorTransaction} The transaction object.
+ */
+export const parseAnchorTransaction = (
+  transaction: string,
+): AnchorTransaction => {
+  const parsed = JSON.parse(transaction);
+  if (parsed.kind === "withdrawal") {
```

Comment:
hmm I like the idea but imo I think separating the parse functions would be better bc of the types. The types for sep6 and sep24 transactions are similar but have slight differences. I think if we'd want to add a parse method for sep6 it should be a separate method to make it clear what the method is returning 🤔  

right now it's returning `AnchorTransaction`, if it was returning `AnchorTransaction | Sep6Transaction` I think it would be a bit messy / too general

imo we can add in a separate PR if it's needed

if you feel strongly though lmk!

Reply:
that makes sense to me!